### DISCUSSION TITLE
Continues chat

### DISCUSSION THREAD

**Discussion by guyzilb** [vanna-ai/vanna#414] [Q&A]
Is it possible to create a chat session and not ask a new question every time?
I want to be able to create a conversation and not ask new questions every time

**Comment by zainhoda** [comment]
It's technically possible (presuming there's enough tokens left in the context window) but we've found that open-ended conversation continuation frustrates the end-users more than having to re-phrase a question.

For argument's sake, let's game it out. What's an example of a question that a user asks and then what's an example of how the conversation continues?

**Comment by guyzilb** [comment]
1. if he generates for me wrong SQL and I just want to tell him to make some adjustments, to 
make it accurate 
2. someone who didn't know SQL and wants now to change the parameter 
I don't want every time ask the question again. 


Is there a department with unusually high expenses?

what is unusual? he can generate a query  and decide the average is unusual but then I want to check 10% is unusual and then 15 
and see the differences
